src/main/java/org/apache/pulsar/io/jcloud/sink/BlobStoreAbstractSink.java

```diff
@@ -115,6 +122,21 @@ public void open(Map<String, Object> config, SinkContext sinkContext) throws Exc
         this.blobWriter = initBlobWriter(sinkConfig);
     }
 
+    private Map<String, String> parseTopicsToPathMappingString(String topicsToPathMappingString) {
+        Map<String, String> topicsToPathMapping = new HashMap<>();
+        if (StringUtils.isNotEmpty(topicsToPathMappingString)) {
+            for (String topicPathPair : topicsToPathMappingString.split(",")) {
+                String[] topicToPathMapping = topicPathPair.split("=");
+                if (topicToPathMapping.length == 2) {
+                    topicsToPathMapping.put(topicToPathMapping[0], topicToPathMapping[1]);
```

Comment:
maybe it's better to print the final mapping here

Reply:
done